evaluation_service/src/main.rs

```diff
@@ -1,80 +1,583 @@
+use std::io;
+use std::path::PathBuf;
+
 use protos::service::evaluation::{evaluation_server::*, *};
 use protos::utils::*;
 use tonic::{transport::*, Request, Response, Status};
+use utils::storage::FsStorageHelper;
+
+const ROOT_PATH: &str = "/evaluation_files";
+const SERIALIZED_EXTENSION: &str = "ser";
+const USER_SCORING_FILE_NAME: &str = "user_scoring";
+const PROBLEMS_FOLDER_NAME: &str = "problems";
+const TESTCASES_FOLDER_NAME: &str = "testcases";
+const EVALUATION_FILES_FOLDER_NAME: &str = "files";
+const INPUT_FILE_NAME: &str = "input";
+const OUTPUT_FILE_NAME: &str = "output";
+const IO_EXTENSION: &str = "txt";
+const PROBLEM_METADATA_FILE_NAME: &str = "metadata";
+
+fn internal_error<T>(e: T) -> Status
+where
+    T: core::fmt::Debug + core::fmt::Display,
```

Comment:
You are only using Debug, maybe you should just have T: Debug, or have T: Display and use {} instead of {:?}